toonz/sources/toonzqt/styleselection.cpp

```diff
@@ -91,7 +91,7 @@ bool pasteStylesDataWithoutUndo(TPalette *palette, TPaletteHandle *pltHandle,
       break;
     }
 
-    if (palette->getStylePage(styleId) < 0) {
+    if (palette->getStylePage(styleId) != NULL) {
```

Comment:
Should it be  `if (palette->getStylePage(styleId) == NULL) {` ?

I google-translated the Italian comment below.
If this condition is true, then the comment says; `styleId is not used: I use that. (cut / paste used to move styles)`
Whereas if this condition is false, the comment says; `styleId is already used. I have to take another one` .